share/SEDs/getSEDs.py

```diff
@@ -16,23 +16,27 @@
 #    and/or other materials provided with the distribution.
 #
 """@file getSEDs.py
-Grab example SEDs from the web, clip them at 22050 Angstroms, and then thin with rel_err = 1.e-5 or
-1.e-3 (two versions).
-Note that the outputs of this script, which are the files GALSIM_DIR/share/CWW_*.sed, are
+Grab example SEDs from the web, clip them and then thin.
+Note that the outputs of this script, which are the files GALSIM_DIR/share/SEDs/CWW_*.sed, are
```

Comment:
These scripts are mostly just for provenance; I don't expect users to run them.  (I had already added the downloaded data to the repo).  I thought they'd be more discoverable if they lived next to the data, though, which is why they're in share/.